apps/api/src/modules/summary/summary.service.ts

```diff
@@ -24,19 +25,25 @@ export class SummaryService extends CoreService<Summary> {
 
   async createSummary(createSummaryInput: CreateSummaryDTO): Promise<Summary> {
     const { inputFile } = createSummaryInput;
+    
+
     if (inputFile) {
       const { createReadStream, filename } = await inputFile;
 
-      const uniqueIdentifier = uuidv4();
+      const uniqueIdentifier = uuidv4().replace(/-/g, '').substring(0, 10);
       const modifiedFilename = `${uniqueIdentifier}_${filename}`;
-      const fileLocation = join(process.cwd(), `./src/upload/${modifiedFilename}`);
+      const fileLocation = join(process.env.Upload_Path, modifiedFilename);
+
+      await fs.ensureDir(process.env.Upload_Path);
 
       return new Promise((resolve, reject) => {
         createReadStream()
           .pipe(createWriteStream(fileLocation))
           .on('finish', async () => {
             const summary = this.summaryRepository.create({ inputFile: modifiedFilename });
+
             try {
+              this.logger.log('Saving Uploaded File Name...');
```

Comment:
Should be  this.logger.log('Saving Uploaded File details...');

Reply:
Updated